source/MaterialXTest/Runtime.cpp

```diff
@@ -457,7 +459,21 @@ TEST_CASE("Runtime: CoreIo", "[runtime]")
         texcoord1_out.connectTo(tiledimage1_texcoord);
         texcoord1_index.getValue().asInt() = 2;
 
-        mx::RtFileIo(stage.getObject()).write(stage.getName().str() + "_export.mtlx");
+        mx::RtWriteOptions writeOptions;
+        writeOptions.writeIncludes = true;
+
+        mx::RtFileIo fileIO(stage.getObject());
+        fileIO.write(stage.getName().str() + "_export.mtlx", &writeOptions);
```

Comment:
Without adding dependents, if you wrote out a document and read it back it again it would fail.
The other way to do this is to load everything into 1 materialx document before hand and then read it in.